### PR TITLE
Improve documentation around accessing Glean events

### PR DESCRIPTION
**Pull request by wlach**

### CODE REVIEW

File: src/concepts/pipeline/glean_data.md [hunk 1]
```diff
@@ -43,21 +45,33 @@ browser:
...
-SELECT
-  COUNT(*),
-  COUNTIF(metrics.boolean.browser_is_default)
-FROM
-  -- We give the table an alias so that the table name `metrics` and field name
-  -- `metrics` don't conflict.
-  org_mozilla_fenix.metrics AS m
-WHERE
-  date(submission_timestamp) = '2019-11-11'
-```
```
Comment (wlach) [inline src/concepts/pipeline/glean_data.md:57]: I think it's better to leave the examples in the cookbook section, and have the focus here be on being a reference.

File: src/cookbooks/accessing_glean_data.md [hunk 2]
```diff
@@ -47,3 +48,48 @@ SELECT AVG(metrics.metrics.counter.metrics_top_sites_count)
...
+    submission_timestamp,
+    e.timestamp AS event_timestamp,
+    e.category AS event_category,
+    e.name AS event_name,
+    e.extra AS event_extra
+FROM sample CROSS JOIN UNNEST(sample.events_array) AS e
+WHERE
+    e.category = 'engine_tab'
+    AND e.name = 'foreground_metrics'
+)
```
Comment (fbertsch) [inline src/cookbooks/accessing_glean_data.md:88]: Are you okay with consolidating this?
```suggestion
WITH events AS (
SELECT
    submission_timestamp,
    client_info.client_id,
    event.timestamp AS event_timestamp,
    event.category AS event_category,
    event.name AS event_name,
    event.extra AS event_extra,
FROM org_mozilla_fenix.events AS e
CROSS JOIN UNNEST(e.events) AS event
WHERE
    submission_timestamp = '2021-05-03'
    AND sample_id = 42 -- 1% sample for development
    AND event.category = 'engine_tab'
    AND event.name = 'foreground_metrics'
)
```

**Review by fbertsch — APPROVED** [review]
r+wc

File: src/concepts/pipeline/glean_data.md [hunk 3]
```diff
@@ -43,21 +45,33 @@ browser:
...
+          Number of active tabs (with an engine session assigned) when the app
+          went to the background.
+        ...
+```
+
+In this case the event's `category` would be `engine_tab` and its name would be `foreground_metrics`.
 
-Documentation TBD. [See bug 1606836](https://bugzilla.mozilla.org/show_bug.cgi?id=1606836)
+You can use the record's `timestamp` and `extra` fields to get the event's timestamp and specifics related
+to the event.
```
Comment (wlach) [inline src/concepts/pipeline/glean_data.md:77]:
```suggestion
to the event.
For a complete example, see ["event metrics" under Accessing Glean Data](../../cookbooks/accessing_glean_data.md#event-metrics).
```

**Pull request merged**